app/views/users/_create_form.html.erb

```diff
@@ -21,7 +21,7 @@
     <div class="row">
       <div class="col-md-6">
         <div class="form-group" id="username_div">
-          <label for="username"><%= translation('user_sessions.new.username') %></label>
+          <label for="username-signup"><%= translation('user_sessions.new.username') %></label>
```

Comment:
Just checking if this has affected our client-side validation at all? I would guess not but good to note this so people feel secure about it. Thank you!!!